src/ngraph/frontend/onnx_import/onnx.hpp

```diff
@@ -18,69 +18,65 @@
-        /// \param[in]  op_name  The ONNX operator name.
-        /// \param[in]  version  The ONNX operator set version.
-        /// \param[in]  domain   The domain the ONNX operator is registered to.
+        /// \param[in]  op_name  The ONNX operator name
+        /// \param[in]  version  The ONNX operator set version
+        /// \param[in]  domain   The domain the ONNX operator is registered to
+        ///                      If not set default domain "ai.onnx" is used
         ///
         /// \return     True if operator is supported, False otherwise.
-        ///
         NGRAPH_API
         bool is_operator_supported(const std::string& op_name,
                                    std::int64_t version,
                                    const std::string& domain = "ai.onnx");
 
-        /// \brief Convert an ONNX model to nGraph function
-        /// The function translated serialized ONNX model to nGraph function. The serialized
-        /// ONNX model is read from input stream.
-        /// \param sin       input stream (e.g. file stream, memory stream, etc)
-        /// \return The function returns a nGraph function representing single output from graph.
+        /// \brief      Import and convert an ONNX model from stream to nGraph function.
+        ///             The function translated serialized ONNX model to nGraph function.
+        ///             The serialized ONNX model is read from input stream.
+        ///
+        /// \note       If parsing stream fails or ONNX model contains not supported ops
+        ///             ngraph_error exception can be thrown.
+        ///
+        /// \param[in]  stream      The input stream (e.g. file stream, memory stream, etc)
+        ///
+        /// \return     The function returns a nGraph function representing single output from graph
         NGRAPH_API
-        std::shared_ptr<Function> import_onnx_model(std::istream& sin);
+        std::shared_ptr<Function> import_onnx_model(std::istream& stream);
 
-        /// \brief Convert an ONNX model to nGraph functions
-        /// The function translated serialized ONNX model to nGraph functions. The ONNX model
-        /// is read from ONNX file.
-        /// \param filename  file name (relative or absolute path name)
-        /// \return The function returns a nGraph function representing single output from graph.
+        /// \brief     Import and convert an ONNX model from file to nGraph function.
+        ///            The function translated serialized ONNX model to nGraph functions.
+        ///            The ONNX model is read from ONNX file.
+        ///
+        /// \note      If parsing file fails or ONNX model contains not supported ops
+        ///            ngraph_error exception can be thrown.
+        ///
+        /// \param[in] file_path  The path to file containing ONNX model (relative or absolute)
+        ///
+        /// \return    The function returns a nGraph function representing single output from graph
```

Comment:
an nGraph function representing ...

Reply:
I write "The function returns a nGraph Function which represents single output from the created graph", is OK?